test/bases/renku_data_services/data_api/test_sessions.py

```diff
@@ -277,6 +277,7 @@ async def test_post_session_launcher(sanic_client: SanicASGITestClient, user_hea
     assert res.json.get("environment_kind") == "container_image"
     assert res.json.get("container_image") == "some_image:some_tag"
     assert res.json.get("environment_id") is None
+    assert res.json.get("resource_class_id") is None
```

Comment:
it would be nice to test that setting this to a value also works, though that's a bit more difficult as we don't have a fixture to have a pool/class available.

It'd be nice to have something like https://github.com/SwissDataScienceCenter/renku-data-services/blob/main/test/bases/renku_data_services/data_api/test_resource_pools.py#L68-L74 as a fixture that you can depend on. Let me know if you need help with that.

Reply:
sure, I added the fixture to create a resource pool and the test to validate if the user has access to the resource class when creating or updating a launcher.